lib/js/styles/Colors/Colors.stories.ts

```diff
@@ -0,0 +1,26 @@
+import { Args, Meta, StoryFn } from '@storybook/vue';
+
+import ItemsList from './../ItemsList.vue';
+import source from '../../../styles/settings/colors/_raw.json';
```

Comment:
what `source` means here?

Reply:
the same question for other files

Reply:
It means `inputData` have You better name than `listData`?

Reply:
`rawColorsList`?

Reply:
It's fine, updated.